test/dlc_tests/ec2/tensorflow/training/test_tensorflow_training.py

```diff
@@ -36,13 +36,17 @@ def test_tensorflow_standalone_cpu(tensorflow_training, ec2_connection, cpu_only
 @pytest.mark.model("mnist")
 @pytest.mark.parametrize("ec2_instance_type", TF_EC2_GPU_INSTANCE_TYPE, indirect=True)
 def test_tensorflow_train_mnist_gpu(tensorflow_training, ec2_connection, gpu_only):
+    if is_tf1(tensorflow_training):
```

Comment:
It should be better to remain the original approach for tf1 and use the new approach for tf2, as 2 different tests.
We can merge in this one and open another one for recover original tf1 test